File: Emiliano_3er_sem/Java_3er_sem/Leccion1/Mundo_pc/Monitor.java
package Emiliano_3er_sem.Java_3er_sem.Leccion1.Mundo_pc;

public class Monitor {
    private final int idMonitor;
    private String marca;
    private double tamanio;
    private static int contadorMonitores;

    private Monitor(){
        this.idMonitor = ++Monitor.contadorMonitores;   
    }

    public Monitor(String marca, double tamanio){
        this(); //llamando al constructor vacio
        this.marca = marca;
        this.tamanio = tamanio;
    }

    public String getMarca(){
        return this.marca;
    }

    public void setMarca(String marca){
        this.marca = marca;
    }

    public double getTamanio(){
        return this.tamanio;
    }

    public void setTamanio(double tamanio){
        this.tamanio = tamanio;
    }

    public int getIdMonitor(){
        return this.idMonitor;
    }

    @Override
    public String toString(){
        return "Monitor{" + "idMonitor="+ idMonitor +", marca="+ marca + "tamaño="+ tamanio +"}";
    }
}

File: Python3sem/FaltanteSemestreAnterior/Leccion14/mundo_pc/monitor.py
class Monitor:
    contador_monitores = 0


    def __init__(self, marca, tamanio):
        Monitor.contador_monitores += 1
        self._id_monitor = Monitor.contador_monitores
        self._marca = marca
        self._tamanio = tamanio


    def __str__(self):
        return f'id: {self._id_monitor}, marca: {self._marca}, tamaño: {self._tamanio}'


if __name__ == '__main__':
    monitor1 = Monitor('Hp', 15)
    print(monitor1)
    monitor2 = Monitor('Acer', 25)
    print(monitor2)